Add SeleniumSuiteTest and option information (#5)
windup/windup-selenium-tests: README.md

# windup-selenium-tests
*Selenium tests for checking the operation of windup-web ui*

Thanks to [@elisedd](https://github.com/elisedd) for the original version of these tests

> This PR provides code which runs tests against the Web-UI which correspond to the test scripts [here in google docs](https://docs.google.com/spreadsheets/d/1BJGNnAlXPFARxo4Pk6FRiWVQXh-q_ZCQ32E9E_t8f0I/edit?ts=5b98e6c7#gid=814464998).

Reviewer: ```suggestion
This PR provides code which runs tests against the Web-UI which correspond to the test scripts [here in google docs]( https://docs.google.com/spreadsheets/d/11LC8PYUxAmoMngBbFYJebZys69qNNvNxJx8iCadpZLQ/).
```

   - unzip it
   - run `$ ./run_rhamt.sh` script

## Run tests

> ### Run a single test

Reviewer: ```suggestion
### Run a single test
Note there are dependencies between the Tests, documented in the google doc linked above. So only a subset of the tests can be ran in isolation. Each Test has a 2 digit number so replace XX in the command below with the appropriate number e.g. 01
```

   - unzip it
   - run `$ ./run_rhamt.sh` script

## Run tests
### Run a single test

> Note there are dependencies between the Tests, documented in the google doc linked above. So only a subset of the tests can be ran in isolation. Each Test has a 2 digit number so replace XX in the command below with the appropriate number e.g. 01

Reviewer: ```suggestion
Note there are dependencies between the Tests, documented in the google doc linked above. So only a subset of the tests can be ran in isolation. Each Test has a 2 digit number so replace XX in the command below with the appropriate number e.g. 01

```
